index.html

```diff
@@ -0,0 +1,270 @@
+					
+				</div>
+				<div class="btn-allProjects">
+					<button type="button">See Project</button>
+				</div>
+				</div>
+				<div class="post">
+				<div class="post-title">
+					<h1>Proffesional Printing Art</h1>
+					<h1>Printing Data</h1>
+				</div>
+				<div class="post-content">
+					<p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+				</div>
+				<div class="btn-courses">
+					<button type="button" class="function">HTML</button>
+					<button type="button" class="function">bootstrap</button>
+					<button type="button" class="function">Ruby</button>
+					
+				</div>
+				<div class="btn-allProjects">
+					<button type="button">See Project</button>
+				</div>
+				</div>
+				<div class="post">
+					<div class="post-title">
+						<h1>Proffesional Printing Art</h1>
+						<h1>Printing Data</h1>
+					</div>
+					<div class="post-content">
+						<p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+					</div>
+					<div class="btn-courses">
+						<button type="button" class="function">HTML</button>
+					<button type="button" class="function">bootstrap</button>
+					<button type="button" class="function">Ruby</button>
+						
+					</div>
+					<div class="btn-allProjects">
+						<button type="button">See Project</button>
+					</div>
+					</div>
+				
+				
+		</div>
+			<div class="about">
+				<div class="about-content">
+				<h1 class="about-title">About me</h1>
+				<p class="about-description">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+				<button type="button" class="btn btn1 color button" >Get My Resume</button>
+				<hr>
+        <div class="experience">
+			
+					<ul class="exp">
+						<div class="shape"><div class="diamond"></div>
+						<li class="exp1">Language</li></div>
+						   <ul >
+							    <li class="text work">Javascript</li>
+							    <li class="text work">Ruby</li>
+							    <li class="text work">Html</li>
+							    <li class="text work">Css</li>
+						    </ul>
+						<div class="shape"><div class="square"></div>
+						<li class="text exp2">Frameworks</li></div>
+						   <ul>
+							<li class="text work">Bootstrap</li>
+						      <li class="text work">Ruby on Rails</li>
+						      <li class="text work">RSPec</li>
+						      <li class="text work">CapyBara</li>
+						       <li class="text work">Selenium</li>
+						</ul>
+						<div class="shape">
+							<div class="dot"></div>
+							<li class="text exp2">Skills</li>
+						</div>
+					
+						   <ul>
+							    <li class="text work">CodeKit</li>
+						        <li class="text work">Github</li>
+						        <gli class="text work">CodePen</li>
```

Comment:
I would suggest you to change this 'gli' to 'li' because the children of 'ul' are 'li' tags. and 'gli' is wrong and a typo

Reply:
- [Optional] Great Job for updating your comment based on the given feedback👌👍 Kindly Update the previous comment instead of creating a new one as you did for your second comment😂👌
- [Optional] Kindly, uses a couple of emojis and/or gifs in the review. 👍 They make the code review more human-friendly 👽🤖.

Reply:
thank you for highlighting this I am going to continue working on it to get better